packages/atomic/src/components/result-template-components/atomic-result-icon/atomic-result-icon.tsx

```diff
@@ -41,14 +43,27 @@ export class AtomicResultIcon {
     return objectType || fileType || 'custom';
   }
 
-  public render() {
+  private get isCDNIcon() {
+    return !!this.icon?.match(/^https?:\/\//);
+  }
+
+  public async componentWillLoad() {
     const icon = this.icon || this.defaultIcon;
-    const iconPath = getAssetPath(`./assets/${icon}.svg`);
+    const iconPath = this.isCDNIcon
+      ? icon
+      : getAssetPath(`./assets/${icon}.svg`);
+    const response = await fetch(iconPath);
+    this.svg =
+      response.status === 200 || response.status === 304
+        ? await response.text()
+        : null;
+  }
 
+  public render() {
     return (
       <Host
-        class={icon}
-        style={{'background-image': `url(${iconPath})`}}
+        class={!this.isCDNIcon ? this.icon || this.defaultIcon : ''}
+        innerHTML={this.svg}
```

Comment:
I made these changes for two reason:
1. Supporting custom colors on SVGs.
2. Supporting CDN links on SVGs.

Do you believe we should support those?